source/dotnet/Library/AdaptiveCards/ParseContext.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Newtonsoft.Json;
```

Comment:
Looks like this file is missing our copyright header..could you add pls

Reply:
sure

---
In reply to: [340256403](https://github.com/microsoft/AdaptiveCards/pull/3546#discussion_r340256403) [](ancestors = 340256403)